### DISCUSSION TITLE
Version 4.0.6 planned ?

### DISCUSSION THREAD

**Discussion by ghost** [haugene/docker-transmission-openvpn#2884] [General]
Hi,

I just wanted to know if a new version based on Transmission 4.0.6 is planned?
It's been a while since there was an update.

Thanks !

**Comment by gdlx** [comment]
It's done in the dev branch: https://github.com/haugene/docker-transmission-openvpn/commits/dev/

**Reply by SESNut** [reply]
it doesnt look like he pushed it to the docker hub; the dev branch still says 4.0.5
https://hub.docker.com/layers/haugene/transmission-openvpn/dev/images/sha256-f523197319edf2804bd801ae220f4dc8cf7454e54e51a89b7916081037a6d272?context=explore

**Reply by sebnapower** [reply]
That is correct - dev branch that is available via Docker hub is still showing 4.0.5 - can we fix that please - can the DEV built be updated in the hub?

**Reply by benyjr** [reply]
At first I agreed with this comment, but there is a bug in [4.0.6](https://github.com/transmission/transmission/pull/7086) that causes it to over-announce torrents. This resulted in several trackers banning 4.0.6 clients. So the decision not to release a version with 4.0.6 was a good one.

Fortunately, [4.1.0 beta](https://github.com/transmission/transmission/releases/tag/4.1.0-beta.1) was released yesterday which among many fixes, fixed the announce issue.

**Reply by sebnapower** [reply]
I was looking to trying 4.0.6 as with 4.0.5 I am getting a lot CRC errors in downloads like in 90% of them. This is not an issue with my HDDs failing. I switched to QBitTorrent and problem is solved. Would prefer to keep using Transmission. I read that corrupted setting.json file can cause a lot of problems so I have rebuilt container and removed json. Still need to test it but I am not having my hopes up.

**Reply by benyjr** [reply]
@sebnapower 4.1.0 brings a large list of fixes on top of 4.0.6 so I think it's worth waiting for.

**Reply by sebnapower** [reply]
@benyjr lets hope the CRC errors are fixed. I am currently fully switched to qbit but would prefer to come back to Transmission. I am still re-downloading / fixing 160 files that were downloaded with errors...

**Comment by ghost** [comment]
Haugene has updated to version 4.0.6 in dev channel.

https://hub.docker.com/r/haugene/transmission-openvpn/tags

**Comment by haugene** [comment]
Hey. A bit of an update here. It is true that the `dev` branch and tag had transmission 4.0.6, but it doesn't anymore. But I plan to release a tag that will have it. I discussed it briefly here: https://github.com/haugene/docker-transmission-openvpn/discussions/2939

We've been building Transmission ourselves and that has caused a bit of work both with large build scripts, long build times, extra pipelines and more complexity than we want. We started doing it to get Transmission v4 that was supposed to fix a memory leak problem we had. But now that 4.0.5 is in the Ubuntu LTS repo then we go back to installing via `apt install`.

The 4.0.6 version is available on newer Ubuntu versions, and I plan to make a new tag that runs on the latest ubuntu while the `latest` tag will continue to run on LTS versions of Ubuntu. We're trying to focus on stability and maintainability and won't do too much to stay on the newest Transmission versions. At least not build it ourselves for a while.
